WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MePage.java

```diff
@@ -0,0 +1,50 @@
+package org.wordpress.android.e2e.pages;
+
+import android.support.test.espresso.ViewInteraction;
+
+import org.wordpress.android.R;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.wordpress.android.support.WPSupportUtils.clickOn;
+
+public class MePage {
+    // Labels
+    private static ViewInteraction displayName = onView(withId(R.id.me_display_name));
+    private static ViewInteraction usernameLabel = onView(withId(R.id.me_username));
+
+    // Buttons
+    private static ViewInteraction appSettings = onView(withId(R.id.row_app_settings));
+    private static ViewInteraction logoutButton = onView(withId(R.id.row_logout));
+    private static ViewInteraction logoutConfirmationButton = onView(withId(android.R.id.button1));
+
+    public MePage() {
+    }
+
+    public MePage go() {
+        clickOn(R.id.nav_me);
+        displayName.check(matches(isDisplayed()));
+
+        return this;
+    }
+
+    public MePage verifyUsername(String username) {
+        String atUsername = "@" + username;
+        usernameLabel.check(matches(withText(atUsername)));
+
+        return this;
+    }
+
+    public void openAppSettings() {
+        appSettings.perform(click());
+    }
+
+    public void logout() {
+        logoutButton.perform(click());
```

Comment:
The login/logout test fails for me at this step when I'm testing on a smaller device (I used a Pixel). The test isn't scrolling to the logout button, so it fails because the button isn't on screen. It looks like something like this should fix it:

`logoutButton.perform(scrollTo(), click());`

Reply:
This works perfectly! 🤦‍♂️my bad for not checking on smaller screen sizes here, I kind of deprioritized different screen sizes for a while but I should at least cover small and regular size phones